### PR TITLE
simple duration printout

### PR DESCRIPTION
**Pull request by RiesBen**
This pr brings simple duration timings to the cmd line tools:
- plan_rbfe_network
- plan_rhfe_network
- quickrun

adds this line to the output: `Duration: 0:00:30.134822`
it is a super small code change, that adds a decorator to the cli tools.

### CODE REVIEW

File: openfecli/commands/plan_rbfe_network.py [hunk 1]
```diff
@@ -145,6 +148,9 @@ def plan_rbfe_network(
         folder_path=OUTPUT_DIR.get(output_dir),
     )
 
+    end_time = datetime.now()
```
Comment (IAlibay) [inline openfecli/commands/plan_rbfe_network.py:151]: May I suggest using a decorator for this?

Comment (RiesBen) [reply openfecli/commands/plan_rbfe_network.py:151]: problem is, thenyou can not align it anymore within the output. but maybe that is the way...

Comment (IAlibay) [reply openfecli/commands/plan_rbfe_network.py:151]: Don't we have a utils area?

Comment (RiesBen) [reply openfecli/commands/plan_rbfe_network.py:151]: I think I got an idea

**Review by richardjgowers — CHANGES_REQUESTED** [review]
so currently `write` will print to stdout, but this will make stdout noisy if you're also using that for the actual output of the command

**Review by mikemhenry — CHANGES_REQUESTED** [review]

**Review by richardjgowers — APPROVED** [review]
I opened openfreeenergy/gufe#168 which I think is a better system that @mikemhenry is alluding to.  This is good enough and a nice improvement for 0.7 though

**Pull request merged**